.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug report
+description: Create a report to help us improve (search for duplicates first)
+title: '[BUG]: '
+labels: bug
+assignees: []
+body:
+
+ - type: checkboxes
+   id: operating-system
+   validations:
+     required: true
+   attributes:
+    label: Operating System
+    description: Which operating system have you used and triggered the issue?
+    options:
+      - label: macOS
+      - label: Microsoft Windows
+      - label: Linux
+      - label: Unix
+      - label: Other (Tell us at the Specifications part!)
+
+ - type: textarea
+   id: specifications
+   attributes:
+    label: Details
+    description: To understand the issue, we need some details of your setup & system!
+    placeholder: |
+      MCreator version: [2021.1 / branch:master / ...];
+      Generator type: [forge-1.15.2 / forge-1.16.5 / ...];
+      OS [Windows / Ubuntu / macOS / ...]; (if not already listed above)
+   validations:
+     required: true
+      
+ - type: textarea
+   id: description
+   attributes:
+    label: What happened?
+    description: Tell us clearly and unambiguously what went wrong
+    placeholder: Tell us here about the bug/problem
+   validations:
+    required: true
+     
+ - type: textarea
+   id: reproduction
+   attributes:
+    label: Reproduction
+    description: Try to explain what you did for the bug to occur. A good explanation of steps to reproduce helps us locate the bug faster. If applicable, add screenshots to help explain your problem.
+    placeholder: |
+      Step 1: ...
+      Step 2: ...
+      ...
+   validations:
+     required: true
+      
+ - type: textarea
+   id: workspace
+   attributes:
+    label: Example Workspace
+    description: |
+      If the bug is related to the workspace, always provide a minimal reproducible example workspace, export it to a shareable ZIP file, and attach it to the ticket.
+      
+      Keep in mind your workspace of the project is not minimal reproducible example. Make a workspace only with elements crucial for the bug.
+    placeholder: Insert here
+   validations:
+    required: false
+    
+ - type: textarea
+   id: logs
+   attributes:
+    label: Logs
+    description: |
+      If applicable, include game or build logs (full logs, do not crop them, in text format, screenshots of logs are not acceptable) from the Console tab or terminal and steps to reproduce the bug in your workspace.
+      
+      Longer logs should be posted on [GitHub Gists](https://gist.github.com/) (or other paste service) and linked here.
+    placeholder: https://gist.github.com/...
+   validations:
+    required: false
+    
+ - type: checkboxes
+   attributes:
+    label: Checks
+    description: Some things, that should be checked!
```

Comment:
Maybe you should add another checkbox saying like "I have checked that my problem is not on a Wiki or a Knowledge Base page.".

Reply:
Avoid !'s